ocs_sample_library_preview/DataViews.py

```diff
@@ -22,6 +23,8 @@ def __init__(self, client: BaseClient):
         self.__setPathAndQueryTemplates()
```

Comment:
Aren't we missing a `super()` in here for the `Securable`?

Reply:
super call added